pt-BR/mail/chrome/messenger/AccountWizard.dtd

```diff
@@ -118,5 +118,5 @@
 
 <!-- account wizard -->
 <!ENTITY pbAccountWizard-header-email.label "Email">
-<!ENTITY pbAccountWizard-header-rss.label "RSS News &amp; Blogs">
+<!ENTITY pbAccountWizard-header-rss.label "Notícias RSS e Blogs">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY pbAccountWizard-header-rss.label "Notícias `&amp;` RSS e Blogs">